client/modules/IDE/components/PreviewFrame.jsx

```diff
@@ -212,6 +201,12 @@ class PreviewFrame extends React.Component {
     this.addLoopProtect(sketchDoc);
     sketchDoc.head.insertBefore(consoleErrorsScript, sketchDoc.head.firstElement);
 
+    if (this.props.forceFullWidth) {
```

Comment:
I understand where you're coming from with this, but I'm not sure it's a good idea. It will distort the canvas and I think folks will be really confused as to what's happening. I think ideally, the default canvas size for mobile would be `windowWidth` or `windowHeight` but I'm not sure the details on how to implement that in a way that doesn't feel confusing to users.

Reply:
I think it's best to leave this behavior unimplemented for now while we work out a better strategy. The preview screen will render the sketch with original dimensions until then.